aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +30,82 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
+            include different action items from the user, an icon can represent
+            different types of functionality.
```

Comment:
```suggestion
            different types of functionality. It is important that the context of an icon is clear. In some cases, it may be beneficial to pair icons with text to ensure the purpose of the icon is clear.
```